packages/artifact/src/internal/__mocks__/config-variables.ts

```diff
@@ -45,3 +45,7 @@ export function getRuntimeUrl(): string {
 export function getWorkFlowRunId(): string {
   return '15'
 }
+
+export function getRetentionDays(): string | undefined {
```

Comment:
I would suggest adding a test that uses this mocked value.

The entire POST call to create an artifact is mocked so you can use the existing code to simulate a response from the actions service and add some verification to see if the correct value is going through: https://github.com/actions/toolkit/blob/main/packages/artifact/__tests__/upload.test.ts#L373:L410

Here is an existing test where you would just need to add a new `options` input: https://github.com/actions/toolkit/blob/main/packages/artifact/__tests__/upload.test.ts#L81:L83

Reply:
Good suggestion!  I refactored the code to make unit tests easier.  Now all the logic to compute the retention value is moved to a util function so we can test it independently.